wrappers/android/librealsense/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.intel.realsense.librealsense">
+    <uses-permission android:name="android.permission.CAMERA"/>
```

Comment:
This will force every app to ask for a sensitive permission.
Shouldn't we leave it for the app to decide?